src/data/real/ennreal.lean

```diff
@@ -283,6 +283,18 @@ lemma lt_iff_exists_real_btwn :
   ⟨q, rat.cast_nonneg.2 q0, aq, qb⟩,
 λ ⟨q, q0, qa, qb⟩, lt_trans qa qb⟩
 
+lemma lt_iff_exists_nnreal_btwn :
```

Comment:
It exists under the name `lt_iff_exists_coe`, proved both for `ennreal` and `with_top α`. Feel free to rename.

Reply:
It is not the same: `lt_iff_exists_coe` says that `a < b` iff there exists `p` with `a = ↑p` and `↑p < b` , while here I am claiming that there exists `p` with `a < ↑p` and `↑p < b`.

Reply:
Sorry, it's called `with_top.dense_coe` (the non-trivial arrow).

Reply:
I have renamed it, and turned it into an iff.